site/en/docs/privacy-sandbox/chrome-testing/index.md

```diff
@@ -6,138 +6,184 @@ subhead: >
 description: >
   Test your sites with third-party cookies disabled.
 date: 2023-05-18
-updated: 2023-09-01
+updated: 2023-10-09
 authors:
   - alexandrawhite
   - rowan_m
 ---
 
-To prepare for third-party cookie deprecation, we will be providing Chrome-facilitated testing modes that allow sites to preview how site behavior and functionality works without third-party cookies. This post provides an overview of the testing modes Chrome plans to provide and how to access experiment group labels.
+To prepare for third-party cookie deprecation, we will be providing Chrome-facilitated testing modes that allow sites to preview how site behavior and functionality works without third-party cookies. This guide provides an overview of the testing modes Chrome plans to provide and how to access experiment group labels.
 
 {% Aside %}
 
 This testing focuses on measuring the performance of the Privacy Sandbox relevance and measurement APIs (PS R&M APIs): Attribution Reporting, Protected Audience, Topics, Private Aggregation, Shared Storage, and Fenced Frames.
 
 {% endAside %}
 
-We have worked with the CMA to ensure these testing modes align with the
-testing framework (and timeline) for third parties laid out in its
-[guidance on industry testing](https://www.gov.uk/cma-cases/investigation-into-googles-privacy-sandbox-browser-changes#industry-testing).
-As a result, the CMA anticipates that the results from testing in these modes
-can be used in its assessment of the Privacy Sandbox. 
+We will offer two distinct modes:
 
-We plan to have two modes of Chrome-facilitated testing:
+*   **Mode A:** In Q4 2023 organizations testing the PS R&M APIs can opt in to receive consistent labels on a subset of Chrome browsers to allow for coordinated testing across different testers.
+*   **Mode B:** In Q1 2024 Chrome will globally disable third-party cookies for a portion of Chrome browsers.
 
-* **Mode A**: Ad techs can receive control and experiment labels on a portion
-  of traffic and use these to conduct testing and experiments.
-* **Mode B**: Chrome globally disables third-party cookies for some portion of
-  all Chrome users.
+Both modes will continue through to at least Q2 2024 and where third-party cookies are disabled in Mode B, they will remain disabled through the full phase out of third-party cookies.
```

Comment:
I would say just split the sentence. Adding the and is just making the second half of the sentence more difficult to parse. It also probably doesn't need to repeat the disabled cookies bit, that's odd too. It'd be much more readable if it said the following:

"Both modes will continue through to at least Q2 2024. For the Chrome browsers in Mode B, third-party cookies will remain disabled through their full phase-out."

Reply:
```suggestion
Both modes will continue through to at least Q2 2024. Where third-party cookies are disabled in Mode B, they will remain disabled through the full phase out of third-party cookies.
```

Does this split work? I want to keep the emphasis on cookies that are disabled in Mode B, versus Mode B permanently disabling cookies as it's possible that people in Mode B may change settings.

Reply:
I mean if it has worked for all the other reviewers, sure, I don't actually read the emphasis in this sentence. The possibility of people in mode B changing settings isn't implicit in its current form.

Reply:
I'm going to assume yes, as I've immediately got more updates to do to this article anyway so we can clean up as needed. 🥲